ambry-store/src/main/java/com.github.ambry.store/BlobStoreStats.java

```diff
@@ -45,25 +45,25 @@
   private final DiskIOScheduler diskIOScheduler;
 
   /**
-   * Convert a given nested {@link Map} that stores quota related stats to its corresponding
+   * Convert a given nested {@link Map} of accountId to containerId to valid size to its corresponding
    * {@link StatsSnapshot} object.
    * @param quotaMap the nested {@link Map} to be converted
    * @return the corresponding {@link StatsSnapshot} object.
    */
-  static StatsSnapshot convertQuotaMapToStatsSnapshot(Map<String, Map<String, Long>> quotaMap) {
-    Map<String, StatsSnapshot> firstSubTreeMap = new HashMap<>();
+  static StatsSnapshot convertQuotaToStatsSnapshot(Map<String, Map<String, Long>> quotaMap) {
+    Map<String, StatsSnapshot> accountValidSizeMap = new HashMap<>();
     long totalSize = 0;
     for (Map.Entry<String, Map<String, Long>> entry : quotaMap.entrySet()) {
```

Comment:
can you name this accountEntry and the inner one as containerEntry so that its more readable